src/app/shared/components/confirmation-dialog/confirmation-dialog.component.html

```diff
@@ -6,11 +6,11 @@ <h2 mat-dialog-title>{{title}}</h2>
         <span [mat-dialog-close]="false" class="material-icons c-close-dialog-icon">close</span>
     </div>
 </div>
-<div class="c-dialog-body">
+<div class="c-dialog-body" [class.dark]="theme === DialogColorTheme.DARK">
```

Comment:
Just a suggestion: To easily accommodate other color schemes, the `[class.dark]` attribute could be set as `[ngClass]="theme.toLowerCase()"`.

Reply:
Done